app/forms/idv/api_image_upload_form.rb

```diff
@@ -279,6 +282,13 @@ def acuant_sdk_upgrade_ab_test_data
       }
     end
 
+    def getting_started_ab_test_analytics_bucket
+      {
+        getting_started_ab_test_bucket:
```

Comment:
can we document these in `analytics_events.rb`?

Reply:
Added, even though these are intended to be temporary.